app/main.py

```diff
@@ -1 +1,13 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    if len(command) > 0:
+        command_name, file_name, path_to_new_file = command.split()
+        if command_name == "mv" and os.path.isfile(file_name):
+            new_file = os.path.dirname(path_to_new_file)
+            if new_file:
+                if not os.path.exists(new_file):
+                    os.makedirs(new_file)
```

Comment:
This is not needed anymore
Move `os.makedirs(new_file, exist_ok=True)` inside `if block`

`exist_ok` parameter automatically checks if the directory exists, so you don't need to perform the check manually